src/wanna/core/services/pipeline.py

```diff
@@ -250,6 +243,14 @@ def _compile_one_instance(self, pipeline: PipelineModel) -> Path:
             fallback_project_network=self.config.gcp_profile.network,
         )
 
+        labels = {
+            "wanna_name": pipeline.name,
```

Comment:
Please let's change it to `wanna_resource_name` as thinking about it `wanna_name` is to generic